plugins/postcss-hwb-function/INSTALL.md

```diff
@@ -0,0 +1,164 @@
+# Installing PostCSS Hwb Function
+
+[PostCSS Hwb Function] runs in all Node environments, with special
+instructions for:
+
+| [Node](#node) | [PostCSS CLI](#postcss-cli) | [Webpack](#webpack) | [Create React App](#create-react-app) | [Gulp](#gulp) | [Grunt](#grunt) |
+| --- | --- | --- | --- | --- | --- |
+
+## Node
+
+Add [PostCSS Hwb Function] to your project:
+
+```bash
+npm install postcss postcss-hwb-function --save-dev
+```
+
+Use it as a [PostCSS] plugin:
+
+```js
+const postcss = require('postcss');
+const postcssHwbFunction = require('postcss-hwb-function');
+
+postcss([
+  postcssHwbFunction(/* pluginOptions */)
+]).process(YOUR_CSS /*, processOptions */);
+```
+
+## PostCSS CLI
+
+Add [PostCSS CLI] to your project:
+
+```bash
+npm install postcss-cli --save-dev
```

Comment:
We need to install the plugin package, right?
```suggestion
npm install postcss-cli postcss-hwb-function --save-dev
```

Reply:
I actually have no idea :)

I would need to check other `postcss-cli` docs in other plugins.

Reply:
This is not mentioned in other `INSTALL` docs so keeping this as is for now.